internal/version/version.go

```diff
@@ -7,7 +7,7 @@ import (
 
 var version = ""
```

Comment:
I don't think this variable is currently set/used?

Might be nice to do something like the following:

```go
var (
    version = ""
    sum = ""
)

func init() {
  	info, ok := debug.ReadBuildInfo()
	if !ok || info.Main.Version == "" {
		version = "unknown"
	}
    version = info.Main.Version
	sum = info.Main.Sum
}
```

This way, we're not fetching the version info every time `GetVersion` is called.

Reply:
This way it's cleaner, thanks!

Reply:
Please note that building from source tarball without any git info still requires setting the version via ldflags.
See
* https://github.com/go-task/task/issues/1709